src/Network/Network/NetworkWatcher/NetworkConfigurationDiagnostic/InvokeNetworkWatcherNetworkConfigurationDiagnosticCommand.cs

```diff
@@ -0,0 +1,132 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
@irrogozh please add the markdown help file for this cmdlet

Reply:
Added